src/comparisons/elements/data/ie11.js

```diff
@@ -0,0 +1,6 @@
+// get all data-* values
+el.dataset;
+// get single value
+el.dataset.foo;
+// set single value
+el.dataset.bar = { value: 42 };
```

Comment:
JQuery performs serialization with data attributes, so we'll want to stringify/parse these values. Otherwise, they get written as `[object Object]`. Something like this should work

```javascript
function data(el, key, value) {
  // Return the full dataset
  if (key === undefined) {
    const toReturn = {};
    for (const [dKey, dValue] of Object.entries(el.dataset)) {
      try {
        toReturn[dKey] = JSON.parse(dValue);
      } catch (_) {
        // Do nothing
      }
    }
    return toReturn;
  }
  // "key" is a string, either get or set
  else if (typeof key === "string") {
    if (value === undefined) {
      try {
        return JSON.parse(el.dataset[key]);
      } catch (_) {
        return undefined;
      }
    } else {
      el.dataset[key] = JSON.stringify(value);
    }
  }
  // "key" can be any object, writes each key to dataset
  else if (key && typeof key === "object") {
    for (const [dKey, dValue] of Object.entries(key)) {
      el.dataset[dKey] = JSON.stringify(dValue);
    }
  }
}

// get all data-* values
data(el);
// get single value
data(el, "foo");
// set single value
data(el, "bar", { value: 42 });
```